kafkabp/config.go

```diff
@@ -99,6 +100,12 @@ type ConsumerConfig struct {
 	// or it might make things worse.
 	// You are advised to test before using non-empty rack id in production.
 	RackID RackIDFunc `yaml:"rackID"`
+
+	// Optional. Defaults to 5 seconds.
+	//
+	// This affects how often the kafka consumer writes its offsets back to
+	// the cluster.
+	AutoCommitIntervalSeconds int `yaml:"autoCommitInterval"`
```

Comment:
Should we call the yaml value `autoCommitIntervalSeconds` as well so the unit is clear to someone just looking at the config without having to trace through the code?